htg/users/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
+import re
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse, HttpResponse
+
+from my_settings  import SECRET_KEY, ALGORITHM
+from .models import User
+
+class SignInView(View):
+    def post(self, request):
+
+        data = json.loads(request.body)
+        
+        email    = data.get('email',None)
+        password = data.get('password',None)
+
+        if not User.objects.filter(email=email).exists:
+            return JsonResponse({'message':'NOT EXISTS EMAIL'})
+            
+        user            = User.objects.get(email=email)
+        hashed_password = user.password.encode('utf-8')
+
+        if not bcrypt.checkpw(password.encode('utf-8'), hashed_password):
+            return JsonResponse({'message':'INVALID_USER'},status=401)
+            
+        access_token = jwt.encode({'user_id':user.id}, SECRET_KEY, ALGORITHM)
+        
+        return JsonResponse({'message':'GIVE A ACCESS_TOKEN FOR YOU','SUCCESS':access_token},status=200) 
```

Comment:
```suggestion
        return JsonResponse({'access_token' : access_token},status=200) 
```
꼭 메세지를 주는 약속을 하지 않았다면 줄 필요 없겠죠?